test/e2e/iptables-vpc-nat-gw/e2e_test.go

```diff
@@ -31,6 +32,7 @@ import (
 const dockerNetworkName = "kube-ovn-vlan"
 const vpcNatGWConfigMapName = "ovn-vpc-nat-gw-config"
 const networkAttachDefName = "ovn-vpc-external-network"
+const vpcNatGwNs = "kube-system"
```

Comment:
Use `KubeOvnNamespace` defined in package `github.com/kubeovn/kube-ovn/pkg/util`.

Reply:
ok